### PR TITLE
sort demo index

### PR DESCRIPTION
**Pull request by vladimir-dudnik**
, refresh models status

### CODE REVIEW

**Review by IRDonch — APPROVED** [review]

File: demos/README.md
Comment (IRDonch) [inline demos/README.md:102]: This demo won't exist after #1744. I think it's best to wait until that PR is merged and then update this.

Comment (vladimir-dudnik) [reply demos/README.md:102]: agree

**Pull request merged**
